migrations/versions/0363_add_index_key_type.py

```diff
@@ -0,0 +1,29 @@
+"""
+
+Revision ID: 0363_add_index_key_type
+Revises: 0362_broadcast_msg_event
+Create Date: 2022-01-06 13:15:23.797332
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0363_add_index_key_type'
+down_revision = '0362_broadcast_msg_event'
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute("""
```

Comment:
Optional:

If we want to prevent an attempt of `CREATE INDEX` in production, I wonder if it is worth running both of these inside an environment specific `if` statement like https://github.com/alphagov/notifications-api/blob/master/migrations/versions/0327_idx_notification_history.py#L18

Note, not sure if the link there was doing the environment specific if statement for protection or not but it might be useful but equally not essential

Reply:
That's an interesting find. I don't think we need to go that far, though:

- Deployments for API are manual and lengthy, so the likelihood of an accident is low.
- I've added "DO NOT MERGE" to the PR, as we will only merge once Production is ready.